src/app/ipv/tests/checkFormRadioButtons.test.ts

```diff
@@ -0,0 +1,48 @@
+import { expect } from "chai";
+import sinon from "sinon";
+import { checkFormRadioButtonSelected } from "../middleware";
+
+describe("checkFormRadioButtonSelected middleware", () => {
+  const testReq = {
+    body: {},
+    params: { pageId: "page-ipv-identity-document-start" },
+    csrfToken: sinon.fake(),
+    session: {
+      currentPage: "page-ipv-identity-document-start",
+      ipvSessionId: "ipv-session-id",
+      save: sinon.fake.yields(null),
+    },
+    log: { error: sinon.fake() },
+  } as any;
```

Comment:
I think it was already mentioned in another one of Joe's PRs, but it'd be nice if there was some builder/factory methods to create test data but this PR was getting quite big